fuse-test-data-gen.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this is a file automatically generated by your Intellij, so I don't think it should be committed to the repo

Reply:
done